packages/teko/tests/src/Tpetra/tTpetraThyraConverter.cpp

```diff
@@ -87,11 +87,11 @@ double compareTpetraMVToThyra(const Tpetra_MultiVector & eX,
                             int indexStart=-1,int indexEnd=-1); */
 double compareTpetraMVToThyra(const Tpetra::MultiVector<ST,LO,GO,NT> & eX,
                             const Teuchos::RCP<const Thyra::MultiVectorBase<ST> > & tX,
-                            int verbosity,std::ostream & os,GO indexStart=-1,GO indexEnd=-1);
+                            int verbosity,std::ostream & os,int indexStart=-1,GO indexEnd=-1);
```

Comment:
See above: use `Teuchos::OrdinalTraits<GO>::invalid()` instead of `-1`, and preserve `GO` here.

Reply:
Same comment as above.

Reply:
@eric-c-cyr is OK with this change - in the unit tests, indexStart will never be so large that long will be needed. Therefore keeping change.